### PR TITLE
More informations for maintainers : red flags for code reviews, BC breaks, what it means to approve a PR

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | I continue the work started in https://github.com/PrestaShop/docs/pull/741 by listing red flags for code reviews, how to deal with BC breaks, what it means to approve a PR
| Fixed ticket? |

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]
💪🏻

File: src/content/1.7/project/maintainers-guide/reviewing-pull-requests.md [hunk 1]
```diff
@@ -62,6 +62,47 @@ There are multiple bots that monitor the Issues and Pull Requests on GitHub.
 
 [Prestonbot][prestonbot] will try to add relevant informations on new Pull Requests and evaluate whether there are missing/invalid items. Following the Pull Request template, Prestonbot will add labels on the Pull Requests.
 
+### Lists of red flags
+
+Here is a list of things of items that should not be approved in a Pull Request
```
Comment (matthieu-rolland) [inline src/content/1.7/project/maintainers-guide/reviewing-pull-requests.md:67]:
```suggestion
Here is a list of things that should not be approved in a Pull Request
```

File: src/content/1.7/project/maintainers-guide/reviewing-pull-requests.md [hunk 2]
```diff
@@ -62,6 +62,47 @@ There are multiple bots that monitor the Issues and Pull Requests on GitHub.
...
+
+- Anti-patterns or code that clearly violates software development best practices such as [SOLID][solid] principles.
+- Code that does not complies with current PrestaShop architecture, unless for good reasons (example: some View logic inside the Model layer).
```
Comment (matthieu-rolland) [inline src/content/1.7/project/maintainers-guide/reviewing-pull-requests.md:70]:
```suggestion
- Code that does not comply with current PrestaShop architecture, unless for good reasons (example: some View logic inside the Model layer).
```

File: src/content/1.7/project/maintainers-guide/reviewing-pull-requests.md [hunk 3]
```diff
@@ -62,6 +62,47 @@ There are multiple bots that monitor the Issues and Pull Requests on GitHub.
...
+- Code that scales poorly and/or performs poorly.
+- Code that is very hard to read and consequently less maintainable.
+- Code that ignore one standard PrestaShop usecase (example: logic that only makes sense for EU rules or USA rules).
```
Comment (matthieu-rolland) [inline src/content/1.7/project/maintainers-guide/reviewing-pull-requests.md:73]:
```suggestion
- Code that ignores one standard PrestaShop usecase (example: logic that only makes sense for EU rules or USA rules).
```

File: src/content/1.7/project/maintainers-guide/reviewing-pull-requests.md [hunk 4]
```diff
@@ -62,6 +62,47 @@ There are multiple bots that monitor the Issues and Pull Requests on GitHub.
...
+- Code that ignore part of PrestaShop user audience (example: CSS that is not [RTL][rtl]-compliant).
```
Comment (matthieu-rolland) [inline src/content/1.7/project/maintainers-guide/reviewing-pull-requests.md:74]:
```suggestion
- Code that ignores part of PrestaShop user audience (example: CSS that is not [RTL][rtl]-compliant).
```

File: src/content/1.7/project/maintainers-guide/reviewing-pull-requests.md [hunk 5]
```diff
@@ -62,6 +62,47 @@ There are multiple bots that monitor the Issues and Pull Requests on GitHub.
...
+
+### Maintainers are gatekeepers
+
+Approving a Pull Request is actually a meaningful act. It carries multiple messages:
+
+**1. The submitted code is correct and its quality meets our expectations.**
+
+This is obviously a requirement for the Pull Request to be approved.
+
+**2. The outcome of this Pull Request is desireable.**
```
Comment (matthieu-rolland) [inline src/content/1.7/project/maintainers-guide/reviewing-pull-requests.md:96]:
```suggestion
**2. The outcome of this Pull Request is desirable.**
```

**Review by matthieu-rolland — COMMENTED** [review]
Some typo feedbacks

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks

**Pull request merged**